lib/lotus/cli.rb

```diff
@@ -83,6 +83,7 @@ def new(name = nil)
     method_option :application_base_url, desc: 'application base url',                                      type: :string
     method_option :path,                 desc: 'applications path',                                         type: :string
     method_option :url,                  desc: 'relative URL for action',                                   type: :string
+    method_option :method,               desc: "HTTP method for action. Upper/lower case is ignored. Must be one of GET, POST, PUT, PATCH, DELETE, HEAD, OPTIONS, TRACE.", type: :string, default: 'GET'
```

Comment:
should you retrieve the methods from the verbs constant instead of hardcoding here?